docs/running-on-mesos.md

```diff
@@ -387,6 +387,14 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.webui.url</code></td>
```

Comment:
I would agree if running the dispatcher and running the driver weren't mutually exclusive.  But since you can only run one at a time, I think it makes sense to just have one config var.  It's like the "docker.image" config var.  It can refer to the driver docker image or the executor docker image.